src/DGtal/topology/KhalimskySpaceND.ih

```diff
@@ -1347,6 +1370,30 @@ uIsInside( const Cell & p ) const
 //-----------------------------------------------------------------------------
 template < DGtal::Dimension dim, typename TInteger>
 inline
+bool
+DGtal::KhalimskySpaceND< dim, TInteger>::
+cIsInside( const Point & p, DGtal::Dimension k ) const
+{
+  ASSERT( k < DIM );
+  return this->isDimensionPeriodicHelper( k )
+      || uIsValid( p, k );
```

Comment:
`cIsValid` instead of `uIsValid`.
It seems that the point->cell conversion is implicit...